src/components/apiReference/Object.tsx

```diff
@@ -8,57 +8,51 @@ import { TableOfContentsItem } from "../TableOfContents/TableOfContents";
 import "./reference.scss";
 
 interface ObjectProperty {
-    name: string,
-    description: string
+    name: string;
+    description: string;
 }
 
 interface ObjectData {
-    description: string,
-    properties: ObjectProperty[]
-};
+    description: string;
+    properties: ObjectProperty[];
+}
 
 export default function Object(): JSX.Element {
     const { objectId = "" } = useParams();
     const storedObjectData: ObjectData = JSON.parse(sessionStorage[objectId]);
     const objectDescription: string = storedObjectData.description;
     const properties: ObjectProperty[] = storedObjectData.properties;
 
-    const introductionParagraph: ReactNode = (
-        <div className="paragraph">
-            {RenderStringWithUrl(objectDescription)}
-        </div>
-    );
+    const introductionParagraph: ReactNode = <div className="paragraph">{RenderStringWithUrl(objectId, objectDescription)}</div>;
 
     const propertyParagaphs: ReactNode = (
```

Comment:
Typo that we missed in the previous branch
```suggestion
    const propertyParagraphs: ReactNode = (
```